services/contract_call_local.proto

```diff
@@ -150,3 +156,51 @@ message ContractCallLocalResponse {
      */
     ContractFunctionResult functionResult = 2;
 }
+
+/**
+ * The storage changes to a smart contract's storage as a side effect of the function call.
+ */
+message ContractStateChange {
+
+    /**
+     * The contract to which the storage changes apply to
+     */
+    ContractID contractID = 1;
+
+    /**
+     * The list of storage changes.
+     */
+    repeated StorageChange storageChanges = 2;
+}
+
+/**
+ * A storage slot change description.
+ */
+message StorageChange {
+    /**
+     * The storage slot changed.  Up to 32 bytes, big-endian, zero bytes left trimmed.
+     */
+    bytes slot = 1;
+
+    /**
+     * The value read from the storage slot.  Up to 32 bytes, big-endian, zero bytes left trimmed.
+     *
+     * Because of the way SSTORE operations are charged the slot is always read before being written to.
+     */
+    bytes valueRead = 2;
+
+    /**
+     * What the write mode is, either a read-only access or the updated value.
+     */
+    oneof writeMode {
+        /**
+         * The new value written to the slot.  Up to 32 bytes, big-endian, zero bytes left trimmed.
+         */
+        bytes valueWritten = 3;
+
+        /**
+         * If the value was only read this will be true, in lieu of valueWritten.
+         */
+        bool readOnly = 4;
```

Comment:
This doesn't match the format we agreed upon in the HIP

Reply:
Yep, grabbed the wrong cached files.